lib/WebDAV/BasicAuthBackend.php

```diff
@@ -123,11 +129,6 @@ private function login(string $username, string $password)
 
         $profile = $client->getTokenProfile($token->access_token);
 
-        list($user, $userPassword) = $this->loginService->login($profile);
-
-        $this->userSession->completeLogin($user, [
-            'loginName' => $user->getUID(),
-            'password' => $userPassword,
-        ]);
+        $this->loginService->login($profile);
```

Comment:
Just set up basic authentication locally and found this to not be working:

```xml
<?xml version="1.0" encoding="utf-8"?>
<d:error xmlns:d="DAV:" xmlns:s="http://sabredav.org/ns">
  <s:exception>ArgumentCountError</s:exception>
  <s:message>Too few arguments to function OCA\OIDCLogin\Service\LoginService::login(), 1 passed in /var/www/html/custom_apps/oidc_login/lib/WebDAV/BasicAuthBackend.php on line 132 and exactly 3 expected</s:message>
</d:error>
```

```suggestion
        $this->loginService->login($profile, $this->userSession, $this->request);
```